docs/03-cli-reference/09-rollback.md

```diff
@@ -0,0 +1,71 @@
+<!--
+title: Serverless Rollback CLI Command
+menuText: Rollback
+description: Rollback the Serverless service to a specific deployment
+layout: Doc
+-->
+
+# Info
+
+Rollback the Serverless service to a specific deployment.
+
+```bash
+serverless rollback
+```
+
+## Options
+- `--timestamp` or `-t` The stage in your service you want to display information about.
```

Comment:
Would be maybe useful to add mark which option is required (like here: https://github.com/serverless/serverless/blob/master/docs/03-cli-reference/04-invoke.md)